javascript/index.js

```diff
@@ -14,15 +14,20 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  printSeconds()
+  printMinutes() 
 }
 
 function printMinutes() {
-  // ... your code goes here
+  let minutes = chronometer.twoDigitsNumber(chronometer.getMinutes());
+  document.getElementById('minDec').innerHTML = minutes.charAt(0);
```

Comment:
You already have al DOM element references saved in variables in lines 4 to 14. You can use those variables instead of selecting again the DOM elements. It is better for performance.